extensions/blocks/story/story.php

```diff
@@ -27,6 +31,58 @@ function register_block() {
 }
 add_action( 'init', __NAMESPACE__ . '\register_block' );
 
+/**
```

Comment:
Good point. I think photon could simplify some of the logic here. I'm still not fully convinced this block should depend on jetpack features as it could actually be moved to gutenberg core or somewhere else. For now I think trying to simply reuse the already available versions of an image makes sense.